packages/overlay/src/integrations/sentry/components/traces/TraceList.tsx

```diff
@@ -11,6 +12,21 @@ import { getDuration } from '../../utils/duration';
 import HiddenItemsButton from '../HiddenItemsButton';
 import TraceIcon from './TraceIcon';
 
+function TransactionName({ trace }: { trace: Trace }) {
+  const method = String(
+    trace.rootTransaction?.contexts?.trace.data?.method || trace.rootTransaction?.request?.method || '(GET)',
```

Comment:
@BYK i think we should only show method in trace list when there is `request.method` or `contexts.trace.data.method` present.
we should not set this as default value.

Reply:
ok how about these?

Option 1
![image](https://github.com/user-attachments/assets/a44efc43-54b5-4cdc-ab75-dbb70c9ba55c)


Option 2
![image](https://github.com/user-attachments/assets/7b6e6a33-7647-4d02-90a8-da5430fb4424)

Option 3
![image](https://github.com/user-attachments/assets/57b920d4-eb97-4726-8548-3d7c055c0cdb)